src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.ts

```diff
@@ -702,7 +702,7 @@ export class ListComponent implements OnInit {
 
     private setDefaultSort() {
         this.predicate = ListComponent.SORT_ENTRY_NO_VARIABLE;
-        this.reverse = false;
+        this.reverse = true;
```

Comment:
The naming is counter intuitive. if reverse is true, the expectation is the default sorting is descending. Please update the code that uses this variable.